src/main/java/com/java_academy/App.java

```diff
@@ -0,0 +1,13 @@
+package com.java_academy;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
```

Comment:
Use auto formatting for whole code - once you have opening brace in the same line and next time you have it line below